### DISCUSSION TITLE
nameof get's a bit better in C# 12

### DISCUSSION THREAD

**Discussion by giscus** [linkdotnet/Blog.Discussions#62] [General]
# nameof get's a bit better in C# 12

The nameof operator is a great way to get the name of a variable, type, or member. With C# 12 it's getting even better. Let's see how.

https://steven-giesel.com/blogPost/5b0bcb83-4508-4cde-9299-80940bf4709a?utm_source=csharpdigest&utm_medium=&utm_campaign=1654

**Comment by ghadzhigeorgiev** [comment]
Yep, that's a very nice addition indeed
